src/tic_tac_toe/computer.clj

```diff
@@ -0,0 +1,21 @@
+(ns tic-tac-toe.computer
+  (:require [tic-tac-toe.player :as player]
+            [tic-tac-toe.user-interface :as ui]))
+
+(defn- first-available
+  [board]
+  (loop [i 0
+         board board]
+  (if (= (board i) " ")
+    i
+    (recur (inc i) board))))
```

Comment:
Can this be simpler? I tend to avoid `loop`/`recur` because *most* of the time you can achieve the same thing with `map`, `reduce`, and `filter`

(most, but not always 😄 )

Reply:
[Updated this](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-dfaaf09a263c032d5fdde1a0caa5cd41R7).  I just used keep-indexed yesterday, so it was the perfect time to refactor this.